client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -60,27 +60,56 @@ export default React.createClass( {
 		)
 	},
 
-	footerLink() {
-		let logInUrl = config( 'login_url' ) + '?redirect_to=' + encodeURIComponent( window.location.href );
+	subscribeUserByEmailOnly() {
+		this.setState( { submitting: true } );
+		acceptInvite(
+			Object.assign( this.props.invite, { activation_key: this.props.subscriptionActivationKey } ),
+			( error ) => {
+				if ( error ) {
+					this.setState( { error } );
+				} else {
+					window.location = 'https://subscribe.wordpress.com?update=activate&email=' + encodeURIComponent( this.props.invite.meta.sent_to ) + '&key=' + this.props.subscriptionManagementKey;
+				}
+			}
+		);
+	},
+
+	renderEmailOnlySubscriptionLink() {
+		if ( this.props.invite.meta.role !== 'follower' || ! this.props.subscriptionActivationKey ) {
+			return null;
+		}
+
 		return (
-			<a href={ logInUrl } className="logged-out-form__link">
-				{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }
+			<a onClick={ this.subscribeUserByEmailOnly } className="logged-out-form__link">
+				{ this.translate( 'Or follow by email subscription only.' ) }
 			</a>
 		);
 	},
 
+	renderFooterLink() {
+		let logInUrl = config( 'login_url' ) + '?redirect_to=' + encodeURIComponent( window.location.href );
+		return (
+			<div>
+				<a href={ logInUrl } className="logged-out-form__link">
+					{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }
```

Comment:
I think both links should be in the same translation string in order to give Translators some context. "Or" could be tricky to place/translate.